packages/content-type-remote-attachment/README.md

```diff
@@ -22,6 +22,12 @@ Because XMTP messages can only be up to 1MB in size, we need to store the attach
 
 End-to-end encryption must apply not only to XMTP messages, but to message attachments as well. For this reason, we need to encrypt the attachment before we store it.
 
+### Install the package
+
+```bash
+npm i xmtp-content-type-remote-attachment
+```
+
```

Comment:
a few things:

- the package was renamed to include the xmtp org in the scope. `xmtp-content-type-remote-attachment` => `@xmtp/content-type-remote-attachment`.

- can we add instructions for the 2 other package managers?

like this:

```bash
# npm
npm i @xmtp/content-type-remote-attachment

# yarn
yarn add @xmtp/content-type-remote-attachment

# pnpm
pnpm i @xmtp/content-type-remote-attachment
```

- can we move these instructions to the top after the descriptions?

- can we also add these installation instructions for the reactions content type?